site/_docs/reference.md

```diff
@@ -2785,7 +2785,7 @@ In the following:
 | * | CSC(numeric)                                   | Returns the cosecant of *numeric* in radians
 | * | CSCH(numeric)                                  | Returns the hyperbolic cosecant of *numeric*
 | b | CURRENT_DATETIME([ timeZone ])                 | Returns the current time as a TIMESTAMP from *timezone*
-| m | DAYNAME(datetime)                              | Returns the name, in the connection's locale, of the weekday in *datetime*; for example, it returns '星期日' for both DATE '2020-02-10' and TIMESTAMP '2020-02-10 10:10:10'
+| m | DAYNAME(datetime)                              | Returns the name, in the connection's locale, of the weekday in *datetime*; for example, if you config locale to en, it will return 'Sunday' for both DATE '2020-02-10' and TIMESTAMP '2020-02-10 10:10:10', and if you config locale to zh, it will return '星期日'
```

Comment:
I think an impersonal sentence is preferrable, i.e., without "you"
"For a locale of en it will return 'Sunday'" etc.

Reply:
Thanks for your suggestion, I have modified the description.